tools/test/examples/examples_cmake.json

```diff
@@ -44,107 +68,172 @@
+      },
+      {
+          "name": "mbed-os-example-nfc",
+          "github": "https://github.com/ARMmbed/mbed-os-example-nfc",
+          "sub-repo-example": true,
+          "subs": [
+              "NFC_EEPROM",
+              "NFC_SmartPoster"
+          ],
+          "features" : [],
+          "targets" : ["NUCLEO_F401RE"],
+          "toolchains" : [],
+          "exporters": [],
+          "compile" : true,
+          "export": true,
+          "test" : true,
+          "baud_rate": 9600,
+          "compare_log": [
+              "mbed-os-example-nfc/tests/EEPROM.log",
+              "mbed-os-example-nfc/tests/SmartPoster_noShield.log"
+          ],
+          "auto-update" : true
+      },
+      {
+          "name": "mbed-os-example-blockdevice",
+          "github":"https://github.com/armmbed/mbed-os-example-blockdevice",
+          "sub-repo-example": false,
+          "subs": [],
+          "features" : [],
+          "targets" : ["K64F"],
+          "toolchains" : [],
+          "exporters": [],
+          "compile" : true,
+          "export": true,
+          "test" : true,
+          "baud_rate": 9600,
+          "compare_log": ["mbed-os-example-blockdevice/tests/blockdevice.log"],
+          "auto-update" : true
+      },
+      {
+          "name": "mbed-os-example-kvstore",
+          "github":"https://github.com/ARMmbed/mbed-os-example-kvstore",
+          "sub-repo-example": false,
+          "subs": [],
+          "features" : [],
+          "targets" : ["K64F", "DISCO_L475VG_IOT01A"],
+          "toolchains" : [],
+          "exporters": [],
+          "compile" : true,
+          "export": true,
+          "test" : true,
+          "baud_rate": 9600,
+          "compare_log": ["mbed-os-example-kvstore/tests/kvstore.log"],
+          "auto-update" : true
+      },
+      {
+          "name": "mbed-os-example-crash-reporting",
+          "github":"https://github.com/ARMmbed/mbed-os-example-crash-reporting",
+          "sub-repo-example": false,
+          "subs": [],
+          "features" : [],
+          "targets" : ["K64F"],
+          "toolchains" : [],
+          "exporters": [],
+          "compile" : true,
+          "export": true,
+          "test" : true,
+          "baud_rate": 9600,
+          "compare_log": ["mbed-os-example-crash-reporting/tests/crash-reporting.log"],
+          "auto-update" : true
+      },
+      {
+          "name": "mbed-os-example-attestation",
+          "github": "https://github.com/ARMmbed/mbed-os-example-attestation",
+          "sub-repo-example": false,
+          "subs": [],
+          "features" : [],
+          "targets" : [ "K66F", "NUCLEO_F429ZI"],
+          "toolchains" : [],
+          "exporters": [],
+          "compile" : true,
+          "export": true,
+          "test" : true,
+          "baud_rate": 9600,
+          "compare_log": ["mbed-os-example-attestation/tests/attestation.log"],
+          "auto-update" : true
+      }
+    ]
 }
+
+  
```

Comment:
Does this file have funny line endings? GitHub is showing a "No newline at end of file" marker.

Reply:
fixed. Had a space.